### ISSUE TITLE
Write some tests!

### ISSUE THREAD

**Issue by eayoungs** [eayoungs/Timing.is#50]
- [ ] Create a test class
- [ ] Add tests for each function except, `google_oauth2()`; needs to be modularized (#)

**Comment by johncoleman83** [comment]
Is this your #1 priority now?  Are you thinking UnitTests (https://docs.python.org/2/library/unittest.html) or DocTests (https://docs.python.org/2/library/doctest.html), or what?

**Comment by eayoungs** [comment]
Yes, for contributors for sure. This would probably be a good way to get involved. I'm probably going to focus on issue #51 first, just to have a useful mock up on the live site. There is a freelancer's union MeetUp tonight where I'd like to solicit feedback and see what the interest or _"buy-in"_ on the concept.

**Comment by eayoungs** [comment]
I was thinking unit tests but i'm more familiar with **py.test**
